daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md

```diff
@@ -1,42 +1,42 @@
 ---
 type: docs
-title: "How-To: Publish a message and subscribe to a topic"
-linkTitle: "How-To: Publish & subscribe"
+title: "How to: Publish a message and subscribe to a topic"
+linkTitle: "How to: Publish & subscribe to topics"
 weight: 2000
 description: "Learn how to send messages to a topic with one service and subscribe to that topic in another service"
 ---
 
-## Introduction
+Now that you've learned what the Dapr Pub/sub building block provides, learn how it can work in your service. The below code example loosely describes an application that processes orders with two services, each with Dapr sidecars:
 
-Pub/Sub is a common pattern in a distributed system with many services that want to utilize decoupled, asynchronous messaging.
-Using Pub/Sub, you can enable scenarios where event consumers are decoupled from event producers.
+- A checkout service using Dapr to subscribe to the topic in the message queue.
+- An order processing service using Dapr to publish a message to RabbitMQ.
 
-Dapr provides an extensible Pub/Sub system with At-Least-Once guarantees, allowing developers to publish and subscribe to topics.
-Dapr provides components for pub/sub, that enable operators to use their preferred infrastructure, for example Redis Streams, Kafka, etc.
 
-## Content Types
+<img src="/images/building-block-pub-sub-example.png" width=1000 alt="Diagram showing state management of example service">
 
-When publishing a message, it's important to specify the content type of the data being sent.
-Unless specified, Dapr will assume `text/plain`. When using Dapr's HTTP API, the content type can be set in a `Content-Type` header.
-gRPC clients and SDKs have a dedicated content type parameter.
+Dapr automatically wraps the user payload in a CloudEvents v1.0 compliant envelope, using `Content-Type` header value for `datacontenttype` attribute. [Learn more about messages with CloudEvents.]({{< ref pubsub-cloudevents.md >}})
 
-## Example:
+The following example demonstrates how your applications publish and subscribe to a topic called `orders`.
 
-The below code example loosely describes an application that processes orders. In the example, there are two services - an order processing service and a checkout service. Both services have Dapr sidecars. The order processing service uses Dapr to publish a message to RabbitMQ and the checkout service subscribes to the topic in the message queue.
+{{% alert title="Note" color="primary" %}}
+ If you haven't already, [try out the new quickstart]({{< ref pubsub-quickstart.md >}}) for a quick walk-through on how to use Pub/Sub.
```

Comment:
```suggestion
 If you haven't already, [try out the pub/sub quickstart]({{< ref pubsub-quickstart.md >}}) for a walk-through on how to use Pub/Sub.
```

Reply:
We should be careful about saying new, since there are only quickstarts and tutorials now